### DISCUSSION TITLE
Welcome to Scala CLI discussions!

### DISCUSSION THREAD

**Discussion by romanowski** [VirtusLab/scala-cli#228] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by bjornregnell** [comment]
Link to Scala CoC: https://scala-lang.org/conduct/
